src/components/Login/Login.js

```diff
@@ -0,0 +1,224 @@
+import React, { useState, useEffect } from 'react';
+import './Login.css';
+import QRREACT from 'react-qr-scanner';
+import { API_CONNECT } from '../../constants/constants';
+
+function Login() {
+  const [username, setUsername] = useState(localStorage.getItem('username') || '');
+  const [password, setPassword] = useState(localStorage.getItem('password') || '');
```

Comment:
heslo nemas preco ukladat do localstorigu, to daj prec, ked najblizsie pridem do Apisu vysvetlim vam podstatu localstorigu ale ocenujem ze si to vobec pouzil